packages/mobile/src/transactions/saga.ts

```diff
@@ -82,6 +88,62 @@ function* cleanupStandbyTransactions({ transactions }: UpdateTransactionsAction)
   }
 }
 
+function* getInviteTransactionDetails(txHash: string, blockNumber: string) {
```

Comment:
yeah i also was thinking about this...but would we then need to query the escrow contract a lot more often? with this client side approach, we only need to query the escrow contract once per invite and we can store the invite details locally for display the next time. but if we want this to be returned in the transaction feed query, would the BE have to fetch this information every time because the api is stateless? maybe @gnardini you have some thoughts on this